SessionNotificationServiceExtension/NotificationServiceExtension.swift

```diff
@@ -154,9 +164,9 @@ public final class NotificationServiceExtension: UNNotificationServiceExtension
                             throw NotificationError.ignorableMessage
                         }
                         
-                        let hasMicrophonePermission: Bool = (AVAudioSession.sharedInstance().recordPermission == .granted)
                         switch ((db[.areCallsEnabled] && hasMicrophonePermission), isCallOngoing) {
                             case (false, _):
+                            Log.info("Microphone permission is \(AVAudioSession.sharedInstance().recordPermission)")
```

Comment:
Do we still want this log? (Since the updated `Permissions` code no longer uses this?)

Reply:
We don't need this anymore, it was a debug log, I'll clean it up. Thanks for catching this!